bitnami/postgresql-ha/templates/postgresql/witness-satefulset.yaml

```diff
@@ -0,0 +1,607 @@
+          securityContext: {{- .Values.volumePermissions.podSecurityContext | toYaml | nindent 12 }}
+          {{- if .Values.volumePermissions.resources }}
+          resources: {{- toYaml .Values.volumePermissions.resources | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: data
+              mountPath: {{ .Values.persistence.mountPath }}
+      {{- end }}
+      {{- if or .Values.witness.initContainers .Values.witness.extraInitContainers }}
+      {{- include "common.tplvalues.render" ( dict "value" (coalesce .Values.witness.initContainers .Values.witness.extraInitContainers) "context" $ ) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: postgresql
+          image: {{ include "postgresql-ha.postgresql.image" . }}
+          imagePullPolicy: {{ .Values.witness.image.pullPolicy | quote }}
+          {{- if .Values.witness.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.witness.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if not .Values.diagnosticMode.enabled }}
+          {{- if .Values.witness.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.witness.lifecycleHooks "context" $) | nindent 12 }}
+          {{- else }}
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                  - /pre-stop.sh
+          {{- end }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.witness.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.witness.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.witness.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.witness.args "context" $) | nindent 12 }}
+          {{- end }}
+          # Auxiliary vars to populate environment variables
+          {{- $postgresqlReplicaCount := int .Values.witness.replicaCount }}
+          {{- $postgresqlFullname := include "postgresql-ha.postgresql" . }}
+          {{- $postgresqlHeadlessServiceName := printf "%s-headless" (include "postgresql-ha.postgresql" .) }}
+          {{- $postgresqlWitnessServiceName := printf "%s-witness" (include "postgresql-ha.postgresql" .) }}
+          {{- $clusterDomain:= .Values.clusterDomain }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.witness.image.debug .Values.diagnosticMode.enabled) | quote }}
```

Comment:
There are some references to values that are no longer defined:
`.Values.witness.image.debug` and `.Values.witness.image.pullPolicy`